packages/packages.csv

```diff
@@ -178,28 +157,16 @@ Microsoft.Extensions.ObjectPool,ship
 Microsoft.Extensions.Options,ship
 Microsoft.Extensions.Options.ConfigurationExtensions,ship
 Microsoft.Extensions.PlatformAbstractions,ship
-Microsoft.Extensions.PlatformAbstractions.Dnx,noship
 Microsoft.Extensions.Primitives,ship
 Microsoft.Extensions.Process.Sources,noship
 Microsoft.Extensions.PropertyActivator.Sources,noship
 Microsoft.Extensions.PropertyHelper.Sources,noship
 Microsoft.Extensions.SecurityHelper.Sources,noship
-Microsoft.Extensions.Testing.Abstractions,ext
 Microsoft.Extensions.TypeNameHelper.Sources,noship
 Microsoft.Extensions.WebEncoders,ship
-Microsoft.IdentityModel.*,ext
 Microsoft.Net.Http.Headers,ship
 Microsoft.Net.Http.Server,ship
 Microsoft.Net.WebSockets.Server,ship
-Microsoft.NETCore.*,ext
 Microsoft.Owin.Security.Interop,ship
-Microsoft.TargetingPack.*,ext
-Microsoft.VisualBasic,ext
 Microsoft.VisualStudio.Web.BrowserLink.Loader,ship
-Microsoft.Win32.*,ext
-NETStandard.Library,ext
-NuGet.*,ext
 PageGenerator,noship
-runtime.*,ext
```

Comment:
We're presorting it now rather than maintaining a list of 3rd party packages. We should modify DependenciesPackages to accept a path for external dependencies instead.